src/style.js

```diff
@@ -233,6 +233,7 @@ const style = css`
     flex-direction: column;
     margin-top: auto;
     width: 100%;
+    padding: 0px;
```

Comment:
Make this a value that can be set, for some reason when I set this here to another value the history bars on top would not show up.  This is the only "impact" on the original library.